src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,95 @@
+<footer class="footer">
+  <div class="container">
+    <div class="row g-4">
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12">
+            <h6 class="mb-4">Categories</h6>
+            <ul class="navbar-nav ">
+              <li *ngFor="let category of categoriesList$ | async" class="nav-item"><a class="nav-link"
+                  [routerLink]="'categories/'+category.id" role="button">{{category.name}}</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4 pb-4">
+          <div class="col-12">
+            <h6 class="mb-4">Stores</h6>
+            <ul class="navbar-nav ">
+              <li *ngFor="let store of storesList$ | async" class="nav-item"><a class="nav-link"
+                  [routerLink]="'stores/'+store.id" role="button">{{store.name}}</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12">
+            <h6 class="mb-4">Get to known us</h6>
+            <ul class="navbar-nav ">
+              <li class="nav-item"><a class="nav-link" role="button">Company</a></li>
+              <li class="nav-item"><a class="nav-link" role="button">About</a></li>
+              <li class="nav-item"><a class="nav-link" role="button">Blog</a></li>
+              <li class="nav-item"><a class="nav-link" role="button">Help Center</a></li>
+              <li class="nav-item"><a class="nav-link" role="button">Our Value</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="border-top py-4">
+      <div class="row align-items-center">
+        <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+          <ul class="list-inline mb-0">
+            <li class="list-inline-item text-dark">Payment Partners</li>
+            <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/amazonpay.svg" alt=""></a>
+            </li>
+            <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/american-express.svg"
+                  alt=""></a> </li>
+            <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/mastercard.svg" alt=""></a>
+            </li>
+            <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/paypal.svg" alt=""></a> </li>
+            <li class="list-inline-item"> <a href="#!"><img src="../assets/images/payment/visa.svg" alt=""></a> </li>
+          </ul>
+        </div>
+        <div class="col-lg-7 mt-4 mt-md-0">
+          <ul class="list-inline mb-0 text-lg-end text-center">
+            <li class="list-inline-item mb-2 mb-md-0 text-dark">Get deliveries with FreshCart</li>
+            <li class="list-inline-item ms-4"> <a href="#!"> <img src="../assets/images/appbutton/appstore-btn.svg"
```

Comment:
only absolute links please

Reply:
You mean changes in : <a href="#!"> because these are links to future features so for now it redirects to the home page

Reply:
i mean the img src